docs/contributing.md

```diff
@@ -31,6 +31,13 @@ And finally install [golangci-lint](https://github.com/golangci/golangci-lint#in
 
 You might want to use an editor such as [Visual Studio Code](https://code.visualstudio.com/download) with the [Go extension](https://code.visualstudio.com/docs/languages/go). Working settings are already in [.vscode/settings.json](../.vscode/settings.json).
 
+## Build and Run
+
+```sh
+go build -o app cmd/updater/main.go
```

Comment:
Maybe make that a `go run` instead? 😉 Also I don't think you need to specify LISTENING_PORT as it defaults to `8000`? If it doesn't it's a bug!

Reply:
you caught me 😆 I am not very familiar with Go, I didn't know about the run command. I updated the file, I am also gonna create an issue for LISTENING_PORT, it does not default to 8000

Reply:
actually if I run `go run cmd/updater/main.go`

I get the following error:

```sh
2021-01-25T21:33:56.240Z        ERROR   open /tmp/go-build849192135/b001/exe/data/config.json: no such file or directory
```

I am using vs code workspaces

Reply:
ah yes good point, let's leave it as you have it then 👍